djaoapp/templates/saas/profile/roles/role.html

```diff
@@ -46,6 +46,18 @@
           </div>
       </div>
   </div>
+  <div class="row mb-2" v-show="items.invited_count > 0 || items.requested_count > 0">
+      <div class="col-12">
+        <div class="btn-group" role="group" aria-label="Basic example" v-cloak>
```

Comment:
Why is there `aria-label="Basic example"` here? Shouldn't it be "group of buttons to show/hide invited and requested roles"?

Reply:
Yeah, totally missed that, my bad.